### ISSUE TITLE
config file error

### ISSUE THREAD

**Issue by KingoftheJiangl** [josephaxisa/henry#7]
According to the [documentation](https://pypi.org/project/henry/#copyright), the config.yml file should be as below. 

hosts:
  host_alias:
    access_token: ''
    host: foo.bar.companyname.com
    id: AbCdEfGhIjKlMnOp
    secret: QrStUvWxYz1234567890

However, auth.py, references the config file using the name 'kwargs', where the value of host is assigned the value 'looker' in cli.py, as such, throwing an error.

host = params['hosts'][kwargs['host']]['host']

I'm surprised no one else has encountered this issue so thought I'd bring it up.

I fixed the issue by finding and replacing all instances of '[kwargs['host']]' with '[host_alias]'.

Second issue I found was maybe because I'm running on a windows machine. In cli.py, line 29 was giving me issues because of file path backslashes, so had to use encode, decode for logging.config to be able to read this correctly.

LOGGING_LOG_PATH = os.path.join(LOGGING_LOG_PATH,'henry.log').encode('unicode_escape').decode()

Thanks, 

Longfei

**Comment by KingoftheJiangl** [comment]
TLDR; replace host_alias with the string of your host alias and pass it as parameter as such:

Hi @josephaxisa Thanks for your comprehensive answers!

In that case I'll change the code back and just replace 'host_alias' with the actual host alias I decide on.

henry pulse --host=prod_looker

Regards, Longfei
